CorsixTH/Lua/entities/machine.lua

```diff
@@ -415,6 +415,7 @@ function Machine:afterLoad(old, new)
       room.hospital:removeHandymanTask(taskIndex, "repairing")
     end
   end
+  self:updateDynamicInfo(true)
```

Comment:
Ugly that the others take string and machine takes bool for the same method name now no?

Reply:
I think it may be because the machine dynamic info is never passed a string, which is why it's not needed.